.gitignore

```diff
@@ -6,6 +6,9 @@ doc/_build/
 dist/
 *.egg-info
 
+# mypy
```

Comment:
I think is something specific from your environment, therefore should not be added here.